workflows/scripts/annotate.py

```diff
@@ -62,7 +61,7 @@ def annotate_cells(
     barcode_platemap_df = pd.read_csv(barcodes_path)
 
     logging.info("Searching plate map name")
-    Path(aggregated_data).name.split("_")[0]
+    plate = pathlib.Path(aggregated_data).stem  # noqa
```

Comment:
what does stem do? curious

Reply:
Steam provides the basename of the file. [source](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem)